common/djangoapps/student/views/management.py

```
@@ -759,6 +759,15 @@ def create_account_with_params(request, params):
     # Track the user's registration
     if hasattr(settings, 'LMS_SEGMENT_KEY') and settings.LMS_SEGMENT_KEY:
         tracking_context = tracker.get_tracker().resolve_context()
+        try:
+            extrainfo_dict = user.extrainfo.__dict__
+            del extrainfo_dict['user_id']
+            del extrainfo_dict['_user_cache']
+            del extrainfo_dict['_state']
+            del extrainfo_dict['id']
+            extrainfo = json.dumps(extrainfo_dict)
+        except Exception:
+            extrainfo = ''
```

@amirtds 

1. Do you know if there is test coverage for the `create_account_with_params` function? I searched the repo for this function and didn't see anything. That's unfortunate if that's the case, but it would take a bit of time to get a test written for it. Bummer.
2. There's a change that `json.dumps` would fail if it finds a field that it could not serialize, so at the least, would you please change the naked exception to `except Exception as as e` and add logging so at least we've got something to report on the error. If not then it could fail silently wit no logging and return an empty string for `extra info`